### PR TITLE
[Chore] coverage

### PR DESCRIPTION
**Pull request by JSHan94**
## Summary of changes

Add testcase for code coverage

## Report of required housekeeping

- [ ] Github issue OR spec proposal link
- [x] Wrote tests
- [ ] Updated API documentation (client/lcd/swagger-ui/swagger.yaml)
- [ ] Added a relevant changelog entry: clog add [section] [stanza] [message]

----

## (FOR ADMIN) Before merging

- [ ] Added appropriate labels to PR
- [ ] Squashed all commits, uses message "Merge pull request #XYZ: [title]" (coding standards)
- [ ] Confirm added tests are consistent with the intended behavior of changes
- [ ] Ensure all tests pass

### CODE REVIEW

File: x/wasm/keeper/msg_server_test.go [hunk 1]
```diff
@@ -114,3 +114,160 @@ func TestMigrateExceedMaxGas(t *testing.T) {
 		NewMsgServerImpl(keeper).MigrateContract(ctx.Context(), types.NewMsgMigrateContract(creator, addr, codeID, []byte(`{"release":{}}`)))
 	})
 }
+
+func TestMsgServerStoreCode(t *testing.T) {
```
Comment (hanjukim) [inline x/wasm/keeper/msg_server_test.go:118]: Please keep naming convention consistency (TestStoreCode)

File: x/wasm/keeper/msg_server_test.go [hunk 2]
```diff
@@ -114,3 +114,160 @@ func TestMigrateExceedMaxGas(t *testing.T) {
...
+	require.NoError(t, err)
+	require.Equal(t, uint64(1), msgRes.CodeID)
+
+	// Verify content
+	storedCode, err := keeper.GetByteCode(ctx, msgRes.CodeID)
+	require.NoError(t, err)
+	require.Equal(t, wasmCode, storedCode)
+}
+
+func TestServerMigrateCode(t *testing.T) {
```
Comment (hanjukim) [inline x/wasm/keeper/msg_server_test.go:142]: TestMigrateCode

**Review by hanjukim — CHANGES_REQUESTED** [review]

File: x/wasm/keeper/msg_server_test.go (hunk 2 above)
Comment (JSHan94) [reply x/wasm/keeper/msg_server_test.go]: Could i change them to `TestStoreCodeMsgServer` and `TestMigrateCodeMsgServer`? There are already `TestStoreCode` and `TestMigrateCode` in contract_test.go file

**Review by hanjukim — APPROVED** [review]

**Comment by hanjukim** [comment]
LGTM

**Pull request merged**